examples/raft/raft.cpp

```
@@ -0,0 +1,484 @@
+#include <grpcpp/create_channel.h>
+#include <grpcpp/security/credentials.h>
+#include <grpcpp/security/server_credentials.h>
+#include <grpcpp/server_builder.h>
+#include <grpcpp/support/status.h>
+
+#include <absl/synchronization/mutex.h>
+#include <absl/time/time.h>
+
+#include <spdlog/spdlog.h>
+
+NodeClient::NodeClient(const ID id, const IP ip)
```

cppcheck (2.10-2)

[performance] Function parameter 'ip' should be passed by const reference.

(passedByValue)